server/testingHelpers.js

```diff
@@ -102,6 +103,7 @@ function sessionFactory(overrides = {}) {
     overrides.respondedAt !== undefined ? overrides.respondedAt : new Date('2021-10-05T20:20:33.000Z'),
     overrides.respondedByPhoneNumber !== undefined ? overrides.respondedByPhoneNumber : null,
     overrides.location !== undefined ? overrides.location : locationFactory(),
+    overrides.isResettable !== undefined ? overrides.isResettable : false,
```

Comment:
If you change the parameter order in `brave-alert-lib`, you'll need to do it here too.